src/typings/global.d.ts

```diff
@@ -192,6 +192,9 @@ declare type BotpressEnvironmentVariables = {
    */
   readonly DISABLE_GLOBAL_SANDBOX?: boolean
 
+  /** Runs all future migrations, ignore completed migrations & sets the config version to the version in package.json */
+  readonly RUNMIG?: boolean
```

Comment:
I would prefer that this variable keep the same prefix (`TESTMIG_`) as the 2 other variables. Then it will be clear that this variable is only for testing migrations.